src/gui/MainWindow.h

```diff
@@ -70,6 +70,10 @@ public slots:
     void hideGlobalMessage();
     void showYubiKeyPopup();
     void hideYubiKeyPopup();
+#ifndef _WIN32
+    void hide();
+    void show();
```

Comment:
These functions are masking Qt base functions and should be `override`. I also dislike changing the ABI based on platform. Instead the `#ifdef Q_OS_WIN` check should be within the function definition.

Reply:
[21:41:11]/opt/buildagent/work/c401303cba1b4098/src/gui/MainWindow.h:73:17: error: only virtual member functions can be marked 'override'
[21:41:11]    void hide() override;

Reply:
Never mind, looks like these are not virtual in Qt. There could be undefined behavior if the base function is called though, might not be a big deal given these are edge case fixes.